### PR TITLE
Change focus to sub accordion on "update benefits" button press

### PR DESCRIPTION
**Pull request by hughes-dustin**
## Description
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/10199

Sets focus to the sub-accordion when  "Update Benefits" button is pressed and reads all updated fields in "Estimated Benefits"

## Testing done
Dev tested

## Acceptance criteria
- [x] Focuses to sub accordion button
- [x] All important fields are wrapped in an `aria-live="polite"` region

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by 1Copenut — APPROVED** [review]
LGTM. Tested the feature branch briefly with Safari + VoiceOver. Will confirm in staging on NVDA and JAWS.

File: src/applications/gi/components/profile/EstimatedBenefits.jsx [hunk 1]
```diff
@@ -105,52 +105,54 @@ export const EstimatedBenefits = ({ profile, outputs, calculator }) => (
...
-        screenReaderSpan={profile.attributes.type === 'ojt' ? month : year}
-        header
-      />
-      <CalculatorResultRow
-        label="Total paid to you"
-        value={outputs.totalPaidToYou.value}
-        bold
-        visible={outputs.totalPaidToYou.visible}
-      />
+    <div aria-live="polite" role="status">
```
Comment (1Copenut) [inline src/applications/gi/components/profile/EstimatedBenefits.jsx:108]: I tried this out using NVDA and Firefox, and was able to recreate the issue BAH QA is seeing with NVDA. We will need to add an `aria-atomic="true"` attribute here. If you'd like to see an updated code mock, check out my codepen here: https://codepen.io/tpierce_402/pen/BajKREW

File: src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx [hunk 2]
```diff
@@ -117,8 +117,8 @@ class EstimateYourBenefitsForm extends React.Component {
       this.setState({ inputUpdated: false });
       this.props.updateEstimatedBenefits();
       setTimeout(() => {
-        scroller.scrollTo(accordionId, getScrollOptions());
-        focusElement(`#${accordionId}`);
+        scroller.scrollTo(accordionButtonId, getScrollOptions());
```
Comment (1Copenut) [inline src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx:120]: I'm not certain we need the scroller behavior here. In testing, it pushes the Estimated Benefits panel off the screen a bit. Can we remove/comment out this line and retest locally before merging?

**Review by 1Copenut — DISMISSED** [review]
@hughes-dustin I've added two requested changes, and will be available to retest promptly the rest of this week. Thank you.

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Review by 1Copenut — APPROVED** [review]
LGTM. I tested the focus handling on localhost with MacOS Safari, Firefox, and Chrome. I'll spot check the screen reader + browser combos on Staging.

**Pull request merged**